fragments/legacy-storage/ceph/README.md

```diff
@@ -0,0 +1,3 @@
+# Ceph Storage
+
+This is the fragment for using leg
```

Comment:
What does this mean?

Reply:
need to finish

Reply:
added a basic readme for now, but will expand.